src/Resources/views/Admin/GiftCardConfiguration/Update/_javascripts.html.twig

```diff
@@ -0,0 +1,6 @@
+<script src="{{ asset('bundles/setonosyliusgiftcardplugin/setono-sylius-gift-card-live-pdf-rendering.js') }}"></script>
```

Comment:
What's the advantage of having a file instead of just having the JS right here?

Reply:
We should never put JS directly into HTML imo. It is not the best practice